clubs/mixins.py

```diff
@@ -0,0 +1,241 @@
+import datetime
+from collections import OrderedDict
+
+from django.core.exceptions import FieldDoesNotExist, MultipleObjectsReturned, ObjectDoesNotExist
+from django.db.models import BooleanField, ManyToManyField
+from rest_framework import serializers
+from rest_framework.renderers import JSONRenderer
+from rest_framework.response import Response
+from rest_framework.utils.serializer_helpers import ReturnDict, ReturnList
+
+
+class ManyToManySaveMixin(object):
```

Comment:
This is really cool! Was thinking about making something like this for a while, think it'll help simplify things.